js/index.js

```diff
@@ -46,3 +46,9 @@ const carrouselImage = document.querySelector(".carrousel__img");
 carrouselArrows.forEach(function(element) {
   element.addEventListener("click", carrousel);
 });
+/* Function about the arrow down or drop down wherever you want
+*/
+function goDown(){
+  let lookdown = document.getElementById('carrousel-section');//take the destini to scroll
```

Comment:
I think you can place the description-comment in the previous line and it will make the code cleaner. What do you think?